Doc/library/typing.rst

```diff
@@ -572,6 +572,32 @@ These can be used as types in annotations and do not support ``[]``.
    * Every type is compatible with :data:`Any`.
    * :data:`Any` is compatible with every type.
 
+.. data:: Never
+
+   Notation for the `bottom type <https://en.wikipedia.org/wiki/Bottom_type>`_,
+   a type that has no members.
+
+   This can be used to define a function that should never be
+   called, or a function that never returns::
+
+     from typing import Never
+
+     def never_call_me(arg: Never) -> None:
+         pass
+
+     never_call_me(1)  # type checker error
+
+     def stop() -> Never:
+         return 1  # type checker error
+
+   The :func:`assert_never()` function uses the ``Never`` type to statically
+   assert that code is unreachable.
```

Comment:
Is that relevant here? What does "statically assert" even mean? (We don't use this phrase elsewhere in this file.) I know that the point is that its definition is
```py
def assert_never() -> Never:
    ...
```
but if the reader doesn't already know that (and why would they if they are reading this?) they wouldn't figure it out from the phrasing here.

Reply:
Removed the mention of `assert_never`.